common/src/main/java/gov/cms/ab2d/common/model/JobProgress.java

```diff
@@ -0,0 +1,39 @@
+package gov.cms.ab2d.common.model;
```

Comment:
Introducing "job slicing" adds considerable complexity to the domain & processing logic with unclear benefits. We are already processing jobs with the maximum degree of concurrency possible (each patient is an async call to BFD), so slices aren't needed here. Creating slices upfront and them managing them is not really needed to create multiple output files, either; I think there are simpler ways to achieve that.

See how logging frameworks roll log files when they reach a certain threshold. They do so transparently in the background without us knowing it, and of course without having us to "slice" streams of log information. This is exactly what I had in mind. We'd need a `JobOutputWriter` class or similar that encapsulates the file writing/rolling logic.